yesod-auth/Yesod/Auth/Message.hs

```diff
@@ -60,6 +60,7 @@ data AuthMessage =
     | ProvideIdentifier
     | SendPasswordResetEmail
     | PasswordResetPrompt
+    | CurrentPassword
```

Comment:
This part is actually a breaking change, and will cause errors in all other translations. I'm OK including it, but we need to include at least a dummy value for the other languages supported by yesod-auth.